index.js

```diff
@@ -17,6 +19,10 @@ const platformLib = (() => {
 
 			return termux;
 		default:
+			if (isWSL) {
```

Comment:
Can you add a comment about what `process.platform` is when using WSL? Just for reference.

Reply:
Sure thing - let me know if it's to your liking :)

Reply:
I guess it wasn't to your liking.

Cheers for getting this merged :)

Reply:
Don't worry. I'm just very particular about how I want things :)

Reply:
haha I get that; I didn't take it personally - was just having a little bit of fun poking the bear ;P

Cheers again for the work you do <3